diff --git a/redirect.admin.inc b/redirect.admin.inc
index 990a22a..cdc4354 100644
--- a/redirect.admin.inc
+++ b/redirect.admin.inc
@@ -192,7 +192,7 @@ function redirect_list_filter_form($keys = '') {
  * Process filter form submission when the Filter button is pressed.
  */
 function redirect_list_filter_form_submit($form, &$form_state) {
-  $form_state['redirect'] = 'admin/config/search/redirect/list/' . trim($form_state['values']['filter']);
+  $form_state['redirect'] = 'admin/config/search/redirect/' . trim($form_state['values']['filter']);
 }
 
 /**
